x-pack/plugins/canvas/server/lib/format_response.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
No `.ts` and test?

Reply:
I think the server routes and related lib files (like this one) should be converted together in a separate PR. I took a stab at it and trying to type the kibana server object is complicated.